statina/parse/batch.py

```diff
@@ -34,13 +34,8 @@ def validate_file_path(file_path: Optional[str]) -> bool:
 
     if not file_path:
         return False
-
     file = Path(file_path)
-
-    if not file.exists():
-        return False
-
-    return True
+    return bool(file.exists())
```

Comment:
file.exists() is already bool

Reply:
It was what pycharm suggested.. But I guess youre right